src/landing-page/components/LandingPage/index.tsx

```diff
@@ -236,29 +244,44 @@ export default function LandingPage(props: LandingPageProps): ReactElement {
     return (
       <ProjectCard key={index}>
         {projectNameElement}
-        <p>{project.description}</p>
-        {publicationElement}
+        <FlexColumn $gap={6}>
+          <p>{project.description}</p>
+          {publicationElement}
+        </FlexColumn>
         {loadButton}
         {datasetList}
       </ProjectCard>
     );
   };
 
+  const [allowMotion, setAllowMotion] = useState(window.matchMedia("(prefers-reduced-motion: no-preference)").matches);
+  useEffect(() => {
+    const mediaQuery = window.matchMedia("(prefers-reduced-motion: no-preference)");
+    mediaQuery.addEventListener("change", () => {
+      setAllowMotion(mediaQuery.matches);
+    });
+    return () => {
+      mediaQuery.removeEventListener("change", () => {
+        setAllowMotion(mediaQuery.matches);
+      });
+    };
+  }, []);
+
```

Comment:
Turns on and off the video playback of the landing page based on accessibility settings in the browser